handlers/accesscontrol/access-control.go

```diff
@@ -181,12 +208,38 @@ func isStale(entry *PlaybackAccessControlEntry) bool {
 	return entry != nil && time.Now().After(entry.MaxAge) && !isExpired(entry)
 }
 
+func (c *HitRecords) cleanUpCache() {
```

Comment:
Shall we put something in to only let this run every X seconds? Otherwise we're running over this locking loop really frequently in a high traffic situation